packages/web-app-files/src/router/spaces.ts

```diff
@@ -15,15 +15,42 @@ export const createLocationSpaces = (name: shareTypes, location = {}): Location
     },
     location
   )
-export const isLocationSpacesActive = isLocationActiveDirector<shareTypes>(
-  createLocationSpaces('files-spaces-personal-home')
+
+export const createLocationSpacesProjects = (name: spaceTypes, location = {}): Location =>
+  createLocation(name, location)
+
+const locationSpacesPersonalHome = createLocationSpaces('files-spaces-personal-home')
+const locationSpacesProjects = createLocationSpacesProjects('files-spaces-projects')
+
+export const isLocationSpacesActive = isLocationActiveDirector<spaceTypes>(
+  locationSpacesPersonalHome,
+  locationSpacesProjects
 )
 
 export const buildRoutes = (components: RouteComponents): RouteConfig[] => [
   {
     path: '/spaces',
     redirect: (to) => createLocationSpaces('files-spaces-personal-home', to)
   },
+  {
```

Comment:
is this needed since we have `/spaces/:namespace`?

Reply:
We didn't manage to make it work this way. Maybe you could have a look?

Reply:
i tested it and have the same problem due BASE_PATH. lets keep it for now as it is